MapboxSearch/sdk/src/main/java/com/mapbox/search/SearchEngineImpl.kt

```diff
@@ -306,6 +310,49 @@ internal class SearchEngineImpl(
         }
     }
 
+    override fun retrieve(mapboxId: String, executor: Executor, callback: SearchResultCallback): AsyncOperationTask {
+        val requestOptions = RequestOptions(
```

Comment:
nit

I'd move such code to external class or just file, in order not to overcomplicate this one.

Reply:
I generally hate utility type class, but I may hate the bloat even worse with these objects.